Database/Archive.ts

```diff
@@ -60,7 +60,8 @@ export class Archive<T = any> extends Silo<T, Archive<T>> {
 		let result: (T & Document) | undefined | ((Document & T) | undefined)[] | ((Document & T)[] & { cursor?: string })
 		if (typeof selection == "string") {
 			const key = await this.getKey(selection)
-			result = key && key.startsWith(this.partitions) ? (await this.backend.doc.get(key))?.value : undefined
+			const document = key && key.startsWith(this.partitions) ? (await this.backend.doc.get(key))?.value : undefined
+			result = document ? document : undefined
```

Comment:
```suggestion
			result = document
```
Unless document can be falsy for other reasons than being undefined